packages/sdk/src/AbstractConfigManager.ts

```diff
@@ -488,7 +493,7 @@ export abstract class AbstractConfigManager {
 
             for (const profile of validationAttempts) {
                 const testValidation: ISshConfigExt = profile;
-                const result = await this.validateConfig(testValidation);
+                const result = await this.validateConfig(testValidation, false);
```

Comment:
if our internal use is mostly calling the function with `false`, should we change the default value to `false` and provide it if needed? 😅 

This may imply that the `validateConfig` function may need to be accessed directly (`private -> protected/public`) by the VSCE 😅